SDBlockDevice/main.cpp

```diff
@@ -0,0 +1,45 @@
+#include "mbed.h"
+#include "SDBlockDevice.h"
+
+// Instantiate the SDBlockDevice by specifying the SPI pins connected to the SDCard
+// socket. The PINS are:
+//     MOSI (Master Out Slave In)
+//     MISO (Master In Slave Out)
+//     SCLK (Serial Clock)
+//     CS (Chip Select)
+SDBlockDevice sd(MBED_CONF_SD_SPI_MOSI, MBED_CONF_SD_SPI_MISO, MBED_CONF_SD_SPI_CLK, MBED_CONF_SD_SPI_CS);
+uint8_t block[512] = "Hello World!\n";
+
+int main()
+{
+    // call the SDBlockDevice instance initialisation method.
```

Comment:
Would recommend consistently using periods at the end of comments or not, rather than a mixture.

Reply:
Same with capitalized first letters of comments.

Reply:
Fixed